libs/tup-components/src/tickets/TicketDetailModal/TicketReplyForm.tsx

```diff
@@ -1,47 +1,58 @@
 import React from 'react';
 
-import { Formik, Form, FormikHelpers } from 'formik';
-import {
-  FormikFileInput,
-  FormikTextarea,
-  FormikSelect,
-} from '@tacc/core-wrappers';
-import { FormGroup } from 'reactstrap';
+import { Formik, Form, FormikHelpers, Field } from 'formik';
+import { FormikFileInput, FormikTextarea } from '@tacc/core-wrappers';
+import { FormGroup, Label } from 'reactstrap';
 import { Button, InlineMessage } from '@tacc/core-components';
-import { useTicketReply } from '@tacc/tup-hooks';
-import * as Yup from 'yup';
+import { Ticket, useTicketReply } from '@tacc/tup-hooks';
 import './TicketModal.global.css';
 
 interface TicketReplyFormValues {
   text: string;
   files: File[];
-  status: string;
+  status: boolean;
 }
 
+/* This validates the form for the first textarea in the form.
+This will also show the red required text underneath the textarea.
+If this is implemented, you must also add the `validationSchema={formSchema}` attribute to the Formik component. 
+***
 const formSchema = Yup.object().shape({
-  text: Yup.string().required('Required'),
+   text: Yup.string().required('Required'),
 });
+***
+*/
 
-export const TicketReplyForm: React.FC<{ ticketId: string }> = ({
-  ticketId,
-}) => {
+export const TicketReplyForm: React.FC<{
+  ticketId: string;
+  ticketData: Ticket | undefined;
```

Comment:
**Question.** Please help. How does `ticketData.Status` differ from `values.status`? I have toruble further reviewing because of my confusion.

Reply:
`ticketData.Status` is the status that is being passed in as a prop from `TicketModal.tsx`
`values.status` is the status that is being passed by the input fields

Reply:
Okay, thanks.

| variable | context | i.e. |
| - | - | - |
| `ticketData.Status` | status before form | status that was |
| `values.status` | status within form | status that will be |